src/sidebar/components/SidebarPanel.tsx

```diff
@@ -10,20 +9,20 @@ import { useSidebarStore } from '../store';
 
 export type SidebarPanelProps = {
   children: ComponentChildren;
-  /** An optional icon name for display next to the panel's title */
-  icon?: IconComponent;
+
   /**
    * A string identifying this panel. Only one `panelName` may be active at any
    * time. Multiple panels with the same `panelName` would be "in sync", opening
    * and closing together.
    */
   panelName: PanelName;
-  title: string;
+
+  /** A description to be set on the panel's `aria-label` */
+  description: string;
   /** Optional callback to invoke when this panel's active status changes */
   onActiveChanged?: (active: boolean) => void;
-  /** What Dialog variant to use */
-  variant?: 'panel' | 'custom';
```

Comment:
I removed the variant as all cases were passing `cutsom` here except for the login one. Instead, I added the full `Card` component there and let consumers decided how the panel content should look like.